docs/lob_guide/src/asciidoc/_chapters/introduction.adoc

```diff
@@ -0,0 +1,152 @@
+////
+/**
+* @@@ START COPYRIGHT @@@
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+*/
+////
+
+[#introduction]
+= Introduction
+
+[#what is lOB]
+== What is LOB
+
+LOB (Large Object), a set of large object data types used to store large volumes of data, provides random and piece-wise access to the data. 
+
+Database support for LOB is not universal.

```

Comment:
In this section, are you trying to define the concept of "LOB" in general (that is, for any database), or just for Trafodion? Are you trying to define the notion of a LOB data type? Or a value that might be stored there?

Reply:
1. Just for Trafodion.
2. Both. Define the notion first, and then the value (1.4 LOB Handle).